setup.py

```diff
@@ -8,13 +8,21 @@
     from setuptools import setup, find_packages
 
 setup(
-    name='crdppf_core',
+    name='crdppf',

```

Comment:
is it mandatory to adapt this name or could we leave crdppf_core to avoid confusion?

Reply:
I would rather call it crdppf_core, because that is the name we are using here https://github.com/sitn/crdppf_core/pull/154/files#diff-6c170fa6d2226afa8682337d014159ccR220 and I think that calling it crdppf would be confusing. If you agree ?

Reply:
Yes I agree absolutely :)